README.md

```diff
@@ -1,19 +1,60 @@
-# WrenEngine
+<p align="center">
+  <a href="https://getwren.ai">
+    <picture>
+      <source media="(prefers-color-scheme: light)" srcset="./misc/wrenai_logo.png">
+      <img src="./misc/wrenai_logo.png">
+    </picture>
+    <h1 align="center">Wren Engine</h1>
+  </a>
+</p>
 
-# How to use?
+<p align="center">
+  <a aria-label="Canner" href="https://cannerdata.com/">
+    <img src="https://img.shields.io/badge/%F0%9F%A7%A1-Made%20by%20Canner-blue?style=for-the-badge">
+  </a>
+  <a aria-label="License" href="https://github.com/Canner/wren-engine/blob/main/LICENSE">
+    <img alt="" src="https://img.shields.io/github/license/canner/WrenAI?color=blue&style=for-the-badge">
+  </a>
+  <a aria-label="Join the community on GitHub" href="https://discord.gg/5DvshJqG8Z">
+    <img alt="" src="https://img.shields.io/badge/-JOIN%20THE%20COMMUNITY-blue?style=for-the-badge&logo=discord&logoColor=white&labelColor=grey&logoWidth=20">
+  </a>
+  <a aria-label="Follow us" href="https://x.com/getwrenai">
+    <img alt="" src="https://img.shields.io/badge/-@getwrenai-blue?style=for-the-badge&logo=x&logoColor=white&labelColor=gray&logoWidth=20">
+  </a>
+</p>
 
-Please refer to the [example](example/README.md) for further details. Start the engine using Docker Compose.
```

Comment:
How about keeping a section for example part? If someone wants to try it without AI, they can refer it.

Reply:
I delete it for a purpose, we can add it back later with our docs, remove for now is better.